<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTools::ClangToolsBasicSettings</class>
 <widget class="QWidget" name="ClangTools::ClangToolsBasicSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="buildBeforeAnalysis">
     <property name="text">
      <string>Build the project before analysis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CppTools::ClangDiagnosticConfigsSelectionWidget" name="clangDiagnosticConfigsSelectionWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CppTools::ClangDiagnosticConfigsSelectionWidget</class>
   <extends>QWidget</extends>
   <header>cpptools/clangdiagnosticconfigsselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
